e2e-tests/upgrade-consistency/run

```diff
@@ -30,178 +30,54 @@ wait_for_sts_generation() {
-    }'
-	wait_for_sts_generation "$cluster-pxc" "2" "1"
-
-	compare_kubectl service/$cluster-pxc "-120"
-	compare_kubectl service/$cluster-proxysql "-120"
-	compare_kubectl statefulset/$cluster-pxc "-120"
-	compare_kubectl statefulset/$cluster-proxysql "-120"
-
-	# test 1.3.0
-	API="pxc.percona.com/v1-3-0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.3.0"}
-    }'
-	wait_for_sts_generation "$cluster-pxc" "3" "1"
-
-	compare_kubectl service/$cluster-pxc "-130"
-	compare_kubectl service/$cluster-proxysql "-130"
-	compare_kubectl statefulset/$cluster-pxc "-130"
-	compare_kubectl statefulset/$cluster-proxysql "-130"
-
-	# test 1.4.0
-	API="pxc.percona.com/v1-4-0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.4.0"}
-    }'
-	wait_for_sts_generation "$cluster-pxc" "4" "1"
-
-	compare_kubectl service/$cluster-pxc "-140"
-	compare_kubectl service/$cluster-proxysql "-140"
-	compare_kubectl statefulset/$cluster-pxc "-140"
-	compare_kubectl statefulset/$cluster-proxysql "-140"
-
-	# test 1.5.0
-	API="pxc.percona.com/v1-5-0"
-	apply_fixed_rbac "v1.5.0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.5.0", "proxysql": { "image": "percona/percona-xtradb-cluster-operator:1.5.0-proxysql" }}
-    }'
-	wait_cluster_consistency "$cluster" 3 2
-	wait_for_sts_generation "$cluster-pxc" "5" "1"
-
-	compare_kubectl service/$cluster-pxc "-150"
-	compare_kubectl service/$cluster-proxysql "-150"
-	compare_kubectl statefulset/$cluster-pxc "-150"
-	compare_kubectl statefulset/$cluster-proxysql "-150"
-
-	# test 1.6.0
-	API="pxc.percona.com/v1-6-0"
-	apply_fixed_rbac "v1.6.0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.6.0"}
-    }'
-	wait_cluster_consistency "$cluster" 3 2
-	wait_for_sts_generation "$cluster-pxc" "6" "1"
-
-	compare_kubectl service/$cluster-pxc "-160"
-	compare_kubectl service/$cluster-proxysql "-160"
-	compare_kubectl statefulset/$cluster-pxc "-160"
-	compare_kubectl statefulset/$cluster-proxysql "-160"
-
-	# test 1.7.0
-	API="pxc.percona.com/v1-7-0"
-	apply_fixed_rbac "v1.7.0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.7.0"}
-    }'
-	wait_cluster_consistency "$cluster" 3 2
-	wait_for_sts_generation "$cluster-pxc" "7" "1"
-
-	compare_kubectl service/$cluster-pxc "-170"
-	compare_kubectl service/$cluster-proxysql "-170"
-	compare_kubectl statefulset/$cluster-pxc "-170"
-	compare_kubectl statefulset/$cluster-proxysql "-170"
-
 	# test 1.8.0
 	API="pxc.percona.com/v1-8-0"
-	apply_fixed_rbac "v1.8.0"
-	kubectl_bin patch pxc "$cluster" --type=merge --patch '{
-        "spec": {"crVersion":"1.8.0"}
-    }'
+	apply_config "$conf_dir/$cluster.yml"
 	wait_cluster_consistency "$cluster" 3 2
-	wait_for_sts_generation "$cluster-pxc" "8" "1"
+	wait_for_sts_generation "$cluster-pxc" "1" "1"
 	compare_kubectl service/$cluster-pxc "-180"
```

Comment:
Could we use generation 8? To save mapping between operator version and generation?
Next release do we need again to change versions? 1.9.0 should become 1 generation and 1.10.0 - 2 one?